en/docs/develop/api-policies/create-a-policy.md

```diff
@@ -0,0 +1,291 @@
+   string|http:HeaderNotFoundError header = res.getHeader(headerName);
+   if (header is http:HeaderNotFoundError) {
+       string message = string `Header ${headerName} is not found`;
+    log:printError(message);
+    return ();
+   }
+   if (header != headerValue) {
+    string validationFailedMessage = string `Header validation failed. Expected ${headerValue} but found ${header}`;
+    log:printError(validationFailedMessage);
+    return ();
+   }
+   return ();
+}
+
+```
+
+In this policy, you are not making any changes to the fault flow stub. Therefore, you can remove the fail flow stub from the policy.bal file.
+
+!!! note
+    The annotations **@mediation:RequestFlow** **@mediation:ResponseFlow** and **@mediation:FaultFlow** are bound with the keywords in the Ballerina.toml. Therefore, the changes you make to the policy stubs should reflect in the Ballerina.toml file. For example, if the policy is applicable only on the request and response paths, the annotation **@mediation:FaultFlow** can be removed from the policy. Then, you **MUST** remove the keyword, **choreo-apim-mediation-fault-flow** from the generated Ballerina.toml file. If failed to do so, the Ballerina compiler will show an error at compile time.
+
+!!! tip 
+    - Organize the source within the default module of the package. Do not add any additional modules.
+    - A policy implementation can contain any combination of flows. A generated project contains stubs for all three flows: request, response, and fault. You can remove any stubs which you do not require. For example, when you create a policy that re-writes the resource paths, you can remove the response and fault stubs. 
+    - The HTTP request/response objects and the context record that gets passed as parameters to the policy functions are passed as references. Therefore, the changes you make to these values are persisted and are passed through all other policies. The request and the response accumulate all the transformations done on them through the attached policies.
+    - The following return type of the policy functions(flows) is unmodifiable:
+        - **http:Response** - Return an HTTP response when you terminate the mediation flow prematurely. For example,  in the in-flow sequence, the mediation sequence would terminate before calling the backend. The mediation policy then sends an HTTP response to the client.
+        - **false** - Return false if you want to terminate the mediation sequence with a pre-defined response(on the Choreo side).
+        - **error** - Return an error if you want to terminate the mediation flow and transfer control to the fault flow. The fault flow would then construct an error response and send it to the client.
+        - **()** - Return () to signal the successful completion of the policy. On completion of the policy, the proxy executes the next policy in the sequence.
```

Comment:
Suggestion:
"Returns `()` to signal the successful completion of the policy. Once the proxy has completed executing one policy, it starts to execute the next policy in the sequence."